mesheryctl/cmd/perf.go

```diff
@@ -86,7 +91,18 @@ var perfCmd = &cobra.Command{
 			log.Print(perfDetails)
 			return
 		}
+		if filePath != "" {
+			var t models.PerformanceSpec
+			err := yaml.Unmarshal([]byte(filePath), &t)
 
+			if err != nil {
+				log.Errorf("Error: File Supplied is invalid.\n%v", err)
+			}
+			testDuration = fmt.Sprintf("%fs", t.EndTime.Sub(t.StartTime).Seconds())
```

Comment:
I suppose we'll want to go through the hassle of verifying the presence of each of these and fail if any required attributes are missing.

Reply:
Nice catch!!! Thanks a lot @leecalcote !!

Reply:
Actually Lee, it would already be tested in the next few lines, such as to check whether URL exist or not:- https://github.com/layer5io/meshery/pull/911/files#diff-95c2c60d8d4612976d9000d256797085L121-L125